### ISSUE TITLE
Form label accessibility error (CAPTCHA?)

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2076]
**Source**

Digital Marin

**Describe the bug**

CAPTCHA label in forms appear to not be accessible. Also the submit button. Seems like it may have something to do with the CAPTCHA?

This happening here:

* https://godigitalmarin.org/feedback
* https://godigitalmarin.org/resources/

Works fine here:

* https://godigitalmarin.org/accessibility-statement/

From Siteimprove:

> A label may already be visible on the page — but to be accessible, that label needs to be associated with the form element in the HTML. This helps visitors using screen readers to understand what information is required, while also making it easier for speech-input users to control the form element

**To Reproduce**
1. Go to https://godigitalmarin.org/resources/ 
2. Run accessibility test
3. See CAPTCHA label error

**Expected behavior**

Field label should be accessible.

**Screenshots**

[Form-field-is-not-labeled-Issues-Accessibility-NextGen-Accessibility.pdf](https://github.com/proudcity/wp-proudcity/files/9123621/Form-field-is-not-labeled-Issues-Accessibility-NextGen-Accessibility.pdf)

![Screen Shot 2022-07-15 at 12 54 55 PM (2)](https://user-images.githubusercontent.com/1569463/179301906-c409039a-a18e-452b-9e5c-8cf7cd1fc7d4.png)

![Screen Shot 2022-07-15 at 12 55 14 PM (2)](https://user-images.githubusercontent.com/1569463/179301914-83576ced-6368-4c54-b0c3-3227f62fa0da.png)
